src/nlp/utils/file_utils.py

```diff
@@ -89,6 +90,38 @@
 INCOMPLETE_SUFFIX = ".incomplete"
 
 
+@contextmanager
+def temp_seed(seed: int, set_pytorch=False, set_tensorflow=False):
```

Comment:
Here there is an error with the tensorflow seed setter that I didn't have time to debug before my time off, maybe you can take a look? You can see the error simply by running the colab at https://colab.research.google.com/drive/1I_B1mcX0cOzOskr0rJN8u8xh_0CIken-?usp=sharing and using `set_tensorflow=True` in its call of `temp_seed`.

Reply:
I fixed tensorflow's rng for eager mode.
Not sure how to do it when eager mode is off. I tried a few things but it didn't work